cloudinstall/core.py

```diff
@@ -68,7 +68,8 @@ def __init__(self, ui, config, loop):
         self.maas = None
         self.maas_state = None
         self.nodes = None
-        self.charm_modules = utils.load_charms()
+        self.charm_modules = utils.load_charms(
+            self.config.getopt('charm_plugins'))
```

Comment:
Not that it's a bug, but for clarity's sake, is this used anywhere?
I can't see it anywhere, and we call load_charms directly later.